Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "ironcore-search-helpers"
-version = "0.1.0"
-authors = ["Colt Frederickson <coltfred@gmail.com>"]
+version = "0.1.1"
+authors = ["IronCore Labs <code@ironcorelabs.com>"]
```

Comment:
Probably doesn't matter, but the other repos I've seen use info@ironcorelabs.com
Is this something that should be consistent?

Reply:
Ya, I think we use both fairly frequently. Since they likely all go to the same place, probably doesn't matter.